terraform/20-app/ecs.service.front-end.tf

```diff
@@ -145,13 +145,6 @@ module "ecs_service_front_end" {
       source_security_group_id = module.front_end_alb.security_group_id
     }
     # egress rules
-    private_api_egress = {
```

Comment:
We don't need this. The FE will make requests to the private API as normal on port 443